### ISSUE TITLE
GAN Loss function is flipped

### ISSUE THREAD

**Issue by vinayjoshi22** [shaohua0116/DCGAN-Tensorflow#5]
https://github.com/shaohua0116/DCGAN-Tensorflow/blob/c49b9801864d60fd81594390c10ce3d95d2b6b7f/model.py#L107

I think, in GAN loss function, For D_LOSS labels for D_REAL should be 1s and for D_FAKE should 0s. For G_LOSS labels for D_FAKE should 1s. Could you please explain on this?
